tst/ProducerClientBasicTest.cpp

```diff
@@ -914,7 +924,7 @@ TEST_F(ProducerClientBasicTest, createStreamStreamUntilTokenRotationStopSyncFree
     EXPECT_EQ(STATUS_SUCCESS, createRealtimeVideoStreamInfoProvider(streamName, TEST_RETENTION_PERIOD, TEST_STREAM_BUFFER_DURATION, &pStreamInfo));
     pStreamInfo->streamCaps.nalAdaptationFlags = NAL_ADAPTATION_FLAG_NONE;
     EXPECT_EQ(STATUS_SUCCESS, createAbstractDefaultCallbacksProvider(TEST_DEFAULT_CHAIN_COUNT,
```

Comment:
Shouldn't we be adding tests for CACHE_TYPE_ALL as well?

Reply:
We do have coverage of the permutations in other tests. These functionality tests target other areas of functionality and permuting them at the higher level will produce a combinatorial explosion. We already have multiple dimensions that we target. Perhaps adding more might not make sense for now.